### PR TITLE
Improved WhatsApp integration - issue #636

### PR DESCRIPTION
**Pull request by ethans**

### CODE REVIEW

File: src/app/model-shared/types.ts [hunk 1]
```diff
@@ -61,12 +61,11 @@ export class PhoneColumn extends radweb.StringColumn {
     if (phone.startsWith('0')) {
       phone = getSettings(context).getInternationalPhonePrefix() + phone.substr(1);
     }
+
     if (phone.startsWith('+'))
       phone = phone.substr(1);
-    if (isDesktop())
```
Comment (noam-honig) [inline src/app/model-shared/types.ts:66]: I've had this before - but I couldn't get this to work with whatsapp desktop on windows.
The advantage of using web.whatsapp - was that with whatsapp web - using this url - it opens always in the same tab - and doesn't kill the previous tabs - which was great.

**Review by noam-honig — CHANGES_REQUESTED** [review]
Thanks for pitching in.

I've commented below.

File: src/app/model-shared/types.ts (hunk 1 above)
Comment (ethans) [reply src/app/model-shared/types.ts:66]: Well, I used WhatsApp Desktop for Windows, and with web.whatsapp.com - it opens the browser version which requests that I stop using the app, and with this, it allows both.
If I have the Desktop app - it uses it, if not - it allows to go to WhatsApp Web.

**Comment by noam-honig** [comment]
I see,

On the other hand - with the link I use - for web.whatsapp - it doesn't
require the extra click when it wants you to start using the app.

Most of my users use the whatsapp web.

I'll reinstall whatsapp desktop and test again.

If you want - I've sent you my phone number on messenger, feel free to call
me.

**Pull request merged**